File: dingo.js
var request = require("request");

var options = { method: 'GET',
    url: 'https://dog.ceo/api/breed/dingo/images/random',
    headers:
        { 'Postman-Token': '780d92bd-0c3d-4395-972f-2915eca5d80c',
            'Cache-Control': 'no-cache' } };

request(options, function (error, response, body) {
    if (error) throw new Error(error);

    console.log(body);
});

File: dingo.py
import requests

url = "https://dog.ceo/api/breed/dingo/images/random"

headers = {
    'Cache-Control': "no-cache",
    'Postman-Token': "39205248-0441-47a8-bbce-e4516cc9d383"
    }

response = requests.request("GET", url, headers=headers)

print(response.text)